string command
string name
string params
---
bool success
string message